crates/core/src/host/wasmer/wasm_instance_env.rs

```diff
@@ -436,15 +460,36 @@ impl WasmInstanceEnv {
         value_len: u32,
         out: WasmPtr<u32>,
     ) -> RtResult<u16> {
-        Self::cvt_ret(caller, "delete_by_col_eq", Call::DeleteByColEq, out, |caller, mem| {
-            let ctx = caller.data().reducer_context();
-            let value = mem.read_bytes(&caller, value, value_len)?;
-            let count = caller
-                .data()
-                .instance_env
-                .delete_by_col_eq(&ctx, table_id.into(), col_id.into(), &value)?;
-            Ok(count.get())
-        })
+        let ctx = caller.data().reducer_context();
+        let txn_type = &ctx.txn_type();
+        let db = &ctx.database();
+        let reducer = &ctx.reducer_name().unwrap_or_default().to_owned();
```

Comment:
The reducer name copies are a bit unfortunate. Are they necessary? Is this what the static metric stuff is about?

https://crates.io/crates/prometheus-static-metric

This is definitely addressable via a followup if so.

Reply:
Ah right, I was thinking about just the abi call names. Yeah, sounds good.